### PR TITLE
translate pod overhead into Chinese

### PR DESCRIPTION
**Pull request by fancc**
translate pod overhead into Chinese

### CODE REVIEW

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+<!--
+When you run a Pod on a Node, the Pod itself takes an amount of system resources. These
+resources are additional to the resources needed to run the container(s) inside the Pod.
+_Pod Overhead_ is a feature for accounting for the resources consumed by the Pod infrastructure
+on top of the container requests & limits.
+-->
+当你在节点上运行一个 Pod 的时候，Pod 自身就要占用掉大量的系统资源。这些资源是运行 Pod 内的容器所需的资源的补充。
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:17]:
```suggestion
当你在节点上运行一个 Pod 的时候，Pod 自身就要占用掉大量的系统资源。这些资源是除运行 Pod 内的容器所需资源之外的资源占用。
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+_Pod 开销_ 是一个特性，用于根据容器请求和限制来计算 Pod 基础结构消耗的资源。
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:18]:
```suggestion
_Pod 开销（Pod Overhead）_ 是一个特性，用于根据容器请求和限制来计算 Pod 基础结构消耗的资源。
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+在 Kubernetes 中，Pod 的开销是根据与 Pod 的 [RuntimeClass](/zh/docs/concepts/containers/runtime-class/) 
+相关的开销在[准入](/zh/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks)时设置的。
+
+<!--
+When Pod Overhead is enabled, the overhead is considered in addition to the sum of container
+resource requests when scheduling a Pod. Similarly, Kubelet will include the Pod overhead when sizing
+the Pod cgroup, and when carrying out Pod eviction ranking.
+-->
+当 Pod 开销被启用，在调度 Pod 时，除了要考虑容器资源请求总和外，还要考虑 Pod 开销。
+同样，Kubelet 在确定 Pod cgroup 的大小以及执行 Pod 驱逐排名时，也将包含 Pod 开销。
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:37]:
```suggestion
同样，Kubelet 在确定 Pod cgroup 的大小以及执行 Pod 驱逐排名时，也将考虑 Pod 开销。
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+<!-->
+## Usage example
+
+To use the PodOverhead feature, you need a RuntimeClass that defines the `overhead` field. As
+an example, you could use the following RuntimeClass definition with a virtualizing container runtime
+that uses around 120MiB per Pod for the virtual machine and the guest OS:
+-->
+## 使用示例
+
+要使用 PodOverhead 特性，你需要一个定义 `overhead` 字段的 RuntimeClass。
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:60]:
```suggestion
要使用 PodOverhead 特性，你需要一个定义了 `overhead` 字段的 RuntimeClass。
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+作为示例，可以在虚拟机和寄宿操作系统中通过一个虚拟化容器运行时来定义 RuntimeClass 如下，
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:61]:
```suggestion
作为示例，可以使用下面定义的包含一个虚拟化容器运行时的 RuntimeClass，
该容器运行时下每个 Pod 将在虚拟机和寄宿操作系统中占用 120 MiB，
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+        cpu: "250m"
+```
+
+<!--
+Workloads which are created which specify the `kata-fc` RuntimeClass handler will take the memory and
+cpu overheads into account for resource quota calculations, node scheduling, as well as Pod cgroup sizing.
+
+Consider running the given example workload, test-pod:
+-->
+创建指定 `kata-fc` RuntimeClass 处理程序的工作负载将会考虑内存和 cpu 开销，
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:83]:
```suggestion
创建时指定了 `kata-fc` RuntimeClass 处理程序的工作负载将会考虑内存和 CPU 开销，
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+
+<!--
+At admission time the RuntimeClass [admission controller](/docs/reference/access-authn-authz/admission-controllers/)
+updates the workload's PodSpec to include the `overhead` as described in the RuntimeClass. If the PodSpec already has this field defined,
+the Pod will be rejected. In the given example, since only the RuntimeClass name is specified, the admission controller mutates the Pod
+to include an `overhead`.
+
+After the RuntimeClass admission controller, you can check the updated PodSpec:
+-->
+在准入阶段，RuntimeClass 的[准入控制器](/docs/reference/access-authn-authz/admission-controllers/)
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:120]:
```suggestion
在准入阶段，RuntimeClass 的[准入控制器](/zh/docs/reference/access-authn-authz/admission-controllers/)
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+将会更新工作负载的 PodSpec 来包含 RuntimeClass 中描述的 `overhead` 字段。如果 PodSpec 中该字段已经被定义，
+该 Pod 将会被拒绝。在下面给出的示例中，由于只指定了 RuntimeClass 名称，所以准入控制器更新了 Pod, 包含了一个 `overhead`。
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:122]:
```suggestion
该 Pod 将会被拒绝。在下面给出的示例中，由于只指定了 RuntimeClass 名称，所以准入控制器更新了 Pod，
使其包含 `overhead`。
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 9]
```diff
@@ -0,0 +1,278 @@
...
+
+在 RuntimeClass 准入控制器后，你可以检查被更新的 PodSpec：
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:124]:
```suggestion
在 RuntimeClass 准入控制器完成其操作后，你可以检查被更新的 PodSpec：
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 10]
```diff
@@ -0,0 +1,278 @@
...
+Once a Pod is scheduled to a node, the kubelet on that node creates a new {{< glossary_tooltip text="cgroup" term_id="cgroup" >}}
+for the Pod. It is within this pod that the underlying container runtime will create containers.
+
+If the resource has a limit defined for each container (Guaranteed QoS or Bustrable QoS with limits defined),
+the kubelet will set an upper limit for the pod cgroup associated with that resource (cpu.cfs_quota_us for CPU
+and memory.limit_in_bytes memory). This upper limit is based on the sum of the container limits plus the `overhead`
+defined in the PodSpec.
+-->
+一旦 Pod 被调度到了某个节点，该节点上的 kubelet 将为该 Pod 新建一个 {{< glossary_tooltip text="cgroup" term_id="cgroup" >}}。
+底层容器运行时将在这个 pod 中创建容器。
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:157]:
```suggestion
底层容器运行时将在这个 Pod 中创建容器。
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 11]
```diff
@@ -0,0 +1,278 @@
...
+
+如果该资源对每个容器都定义了一个限制（定义了受限的 Guaranteed QoS 或者 Bustrable QoS），
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:159]:
```suggestion
如果该资源对每个容器都定义了一个限制（指定了限制值的 Guaranteed QoS 或者 Bustrable QoS），
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 12]
```diff
@@ -0,0 +1,278 @@
...
+kubelet 将会为与该资源（CPU 的 cpu.cfs_quota_us 以及内存的 memory.limit_in_bytes）
+相关的 pod cgroup 设定一个上限。该上限基于容器限制总量与 PodSpec 中定义的 `overhead` 之和。
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:161]:
```suggestion
相关的 Pod cgroup 设定一个上限。该上限基于容器限制总量与 PodSpec 中定义的 `overhead` 之和。
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 13]
```diff
@@ -0,0 +1,278 @@
...
+Check the Pod's memory cgroups on the node where the workload is running. In the following example, [`crictl`](https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md)
+is used on the node, which provides a CLI for CRI-compatible container runtimes. This is an
+advanced example to show PodOverhead behavior, and it is not expected that users should need to check
+cgroups directly on the node.
+
+First, on the particular node, determine the Pod identifier:
+-->
+## 验证 Pod cgroup 限制
+
+检查运行工作负载的节点上 Pod 的内存 cgroup。在以下示例中，节点上将使用具备 CRI 兼容的容器运行时命令行工具 [`crictl`](https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md)，
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:214]:
```suggestion
检查运行工作负载的节点上 Pod 的内存 cgroup。在以下示例中，
将在节点上使用命令行工具 [`crictl`](https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md)，
该工具为 CRI 兼容的容器运行时提供命令行接口。
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 14]
```diff
@@ -0,0 +1,278 @@
...
+这是一个展示 PodOverhead 行为的高级示例，用户无需直接在该节点上检查 cgroup。
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:215]:
```suggestion
这是一个展示 PodOverhead 行为的高级示例，用户一般无需直接在该节点上检查 cgroup。
```

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 15]
```diff
@@ -0,0 +1,278 @@
...
+-->
+执行结果的 cgroup 路径中包含了该 Pod 的 `pause` 容器。Pod 级别的 cgroup 即上面的一个目录。
+```
+        "cgroupsPath": "/kubepods/podd7f4b509-cf94-4951-9417-d1087c92a5b2/7ccf55aee35dd16aca4189c952d83487297f3cd760f1bbf09620e206e7d0c27a"
+```
+
+<!--
+In this specific case, the pod cgroup path is `kubepods/podd7f4b509-cf94-4951-9417-d1087c92a5b2`. Verify the Pod level cgroup setting for memory:
+-->
+在这个例子中，该 pod 的 cgroup 路径是 `kubepods/podd7f4b509-cf94-4951-9417-d1087c92a5b2`。验证内存的 Pod 级别 cgroup 设置：
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:244]:
```suggestion
在这个例子中，该 Pod 的 cgroup 路径是 `kubepods/podd7f4b509-cf94-4951-9417-d1087c92a5b2`。
验证内存的 Pod 级 cgroup 设置：
```

**Comment by ydcool** [comment]
closed #24789

**Comment by tengqm** [comment]
@fancc Please help resolve the conflicts.

**Comment by xichengliudui** [comment]
@fancc

**Comment by tengqm** [comment]
This is done in a different PR.
/close

**Pull request closed without merging**